features/webcryptoapi.yml

```
@@ -1,7 +1,12 @@
-draft_date: 2024-08-12
-name: Web Cryptography API
-description: TODO
-spec: https://w3c.github.io/webcrypto/
+name: Web Cryptography
```

Web Crypto seems to be the more widely used name (e.g., as in docs on MDN). I'll suggest an "also known as" in the description, however, since it seems mixed.

```suggestion
name: Web Crypto
```

---

I wrote up a reasoning for this: https://github.com/web-platform-dx/web-features/pull/1577#issue-2461382789

I didn't see a developer preference either way, and I am concerned "crypto" will forever be confusing. Where are you finding this? I want to make sure I'm looking at the same places. (The whole group should be sharing tips here, since the developer voice part is the fuzziest bit.)

The common pattern I saw in materials (outside of MDN) was a shortname of `webcrypto` and an expanded name of `Web Cryptography API`.

I'm fine putting API back in, as I said in the feature notes.

But I'm not sure the MDN choice of "crypto" here aligns with either developer voice or the optimal disambiguation of terms.

---

resolved per conversation below, ended up back on `Web Cryptography` for now